tests/number/GD_random.gd

```diff
@@ -0,0 +1,48 @@
+extends SimpleTest
+
+func should_return_zero_or_one_when_not_args():
+    var v = GD_.random()
+    expect(v == 0 or v == 1)
```

Comment:
Thanks for writing this unit test! 

These expects dont result in a check yet, it needs to be followed by one of the funcs defined here

```
 expect(v == 0 or v == 1).to.be.truthy()
 expect(...).to.equal(...)
```

Otherwise it wont actually do an equality check 😅

More info [here ](https://github.com/accidentallyc/GodotSimpleUnitTest)

Reply:
Okay, I'll fix it.

Reply:
I'm done! I'm not really familiar with unit testing, so my code might be a bit messy, but at least it gets the job done! <3